Rozdział 17/Hangaman3.html

```diff
@@ -13,8 +13,6 @@
     <script>
         var canvas = document.getElementById("canvas");
         var ctx = canvas.getContext("2d");
-        var letterAppearence = 0;
```

Comment:
to zmienną wywalam całkowicie, jest ona używana tylko wewnętrznie w jednej funkcji